crate2nix/templates/nix/crate2nix/default.nix

```diff
@@ -83,6 +83,7 @@ rec {
         inherit testCrateFlags;
       } ''
         set -ex
+        cd ${crate.src}
```

Comment:
I think that would work for most cases.

There is probably a case where someone relies on stuff that is being generated by the `build.rs` script... In any case this sounds good to go in. We should add a test that makes use of this tho..